templates/workloads/partials/cf/workload-container.yml

```diff
@@ -11,6 +11,13 @@
   DockerLabels:{{range $name, $value := .DockerLabels}}
     {{$name | printf "%q"}}: {{$value | printf "%q"}}{{end}}
 {{- end}}
+{{- if .DependsOn}}
+  DependsOn:
+  {{- range $name, $conditionFrom := .DependsOn}}
+  - Condition: {{$conditionFrom}}
+    ContainerName: {{$name}}
```

Comment:
These yaml templates are super tricky and annoying because they will compile without complaining about formatting, but will fail later. 😤 
These two lines need to be indented by two spaces.

Reply:
Added two extra spaces for each  to this and the sidecars.yml file.